### PR TITLE
added jurisdictions in facility serach and updated popinbox

### PR DESCRIPTION
**Pull request by Swathi-eGov**
## Summary by CodeRabbit

- **New Features**
	- Introduced pagination functionality in the Pop Inbox component for improved data management.
	- Added new properties and methods in the UICustomizations to enhance facility management and user validation.
	- Implemented a button for assigning facilities, triggering a popup for further actions.

- **Bug Fixes**
	- Enhanced validation for mobile numbers to ensure correct formats.

- **Style**
	- Improved layout and spacing of components within the employee microplan inbox.

- **Refactor**
	- Streamlined data handling in various configuration methods for better performance.

### CODE REVIEW

**Review by nabeelmd-eGov — APPROVED** [review]
looks fine to me

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js [hunk 1]
```diff
@@ -4,272 +4,375 @@ import DataTable from "react-data-table-component";
...
+  //show multiple tabs
+  return (
+    <DataTable
+      columns={columns}
+      data={props.censusData}
+      pagination
+      paginationTotalRows={5}
+      selectableRows
+      selectableRowsHighlight
+      onChangeRowsPerPage={handlePerRowsChange}
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js:361]: @Swathi-eGov can you verify whether pagination is working

**Review by nipunarora-eGov — CHANGES_REQUESTED** [review]
1. Pls check comments
2. Pls maintain relevant state and pass them in api calls for pagination(limit,offset)
3. also pls add that showTab(Hiding tab on a status)(PENDING_FOR_VERIFICATION) and don't send assignee on this status. Keep a map for this so later we'll move it to master data

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js [hunk 2]
```diff
@@ -307,7 +318,10 @@ const PopInboxTable = ({ ...props }) => {
     ];
   }, []);
 
-  const handlePageChange = (page) => {};
+  const handlePageChange = (page,totalRows) => {
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js:321]: can you remove these logs

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js [hunk 3]
```diff
@@ -316,7 +330,10 @@ const PopInboxTable = ({ ...props }) => {
     props?.onRowSelect(event);
   };
 
-  const handlePerRowsChange = async (newPerPage, page) => {};
+  const handlePerRowsChange = async (currentRowsPerPage, currentPage) => {
+    console.log(currentRowsPerPage,"currentRowsPerPage")
```
Comment (nipunarora-eGov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js:334]: logs

**Review by nipunarora-eGov — APPROVED** [review]

**Pull request merged**